utils/class-users-query-utils.php

```diff
@@ -128,6 +128,9 @@ private static function build_network_capability_where_clause( $capabilities, $r
 		// Build capability conditions - convert capabilities to roles
 		if ( ! empty( $capabilities ) ) {
 			foreach ( $capabilities as $capability ) {
+				$capability_escaped  = esc_sql( $capability );
```

Comment:
To catch the literal underscores instead of the wildcards:
```suggestion
				$capability_escaped  = esc_sql( $wpdb->esc_like( $capability ) );
```